views/layouts/layout.njk

```diff
@@ -34,6 +34,12 @@
                 {% include "_includes/fip.njk" %}
             </header>
             <main id="content">
+                {% if error %}
```

Comment:
@dsamojlenko : I wound up doing this at the more global layout level instead of the page level since I touched the `flash.js` policy to make this variable available to the view. Since that was a global-ish change and might be useful again, I just went all in. If you think I should move this back to the page level, also fine by me -- let me know and I'll change that up. (And, can do the `res.locals` stuff that I did in `policies/flash.js` in the `MedicalProfessionalController`.)

Reply:
all good! we can use this same method to add `success` messages in the future if the mood should strike.